content/en/docs/ops/ambient/getting-started/index.md

```diff
@@ -111,8 +119,13 @@ $ kubectl apply -f @samples/sleep/notsleep.yaml@
 
 Note: `sleep` and `notsleep` are two simple applications that can serve as curl clients.
 
-Connect `productpage` to the Istio ingress gateway so you can access the bookinfo
-app from outside of the cluster:
+{{< tabset category-name="config-api" >}}
```

Comment:
We shouldn't even document this on ambient IMO

Reply:
I mean "classic"

Reply:
I disagree - nearly all users are still using classic Istio APIs, we don't want to force them to move to new k8s gateway API for ambient.

Reply:
We should push them to - but not force, IMO. We should encourage best practices

Reply:
My first instinct was, same a @howardjohn, that we should only use gateway-api in this doc, since it's all new stuff. But @linsun reminded me that we need to be careful to make it clear that using ambient or not is independent of whether or not one uses gateway-api or classic config. We don't want the getting started with ambient doc to only show users how to do it if they also use an unrelated other new feature (gateway-api). To encourage users (esp new Istio users) to use gateway-api, we should add in this doc the gateway-api boilerplate tip that mentions that the doc shows how to do it with either API, but gateway-api is the future one (like we are currently doing in every other Istio doc using gateways/routes).

Reply:
I think what @frankbu suggested is a reasonable middle ground ^^